code/ds000030.py

```diff
@@ -0,0 +1,85 @@
+"""Load ds000030* data and extract demographic information.
+
+* the UCLA Consortium for Neuropsychiatric Phenomics LA5c Study
+
+Author: Natasha Clarke; last edit 2024-02-14
+
+All input stored in `data/ds000030` folder. The content of `data` is not
+included in the repository.
+
+The data participants.csv is downloaded from https://openneuro.org/datasets/ds000030/versions/00016.
+
+"""
+
+import pandas as pd
+import json
+import argparse
+from pathlib import Path
+
+# Define metadata
+metadata = {
+    "participant_id": {
+        "original_field_name": "participant_id",
+        "description": "Unique identifier for each participant",
+    },
+    "age": {
+        "original_field_name": "age",
+        "description": "Age of the participant in years",
+    },
+    "sex": {
+        "original_field_name": "gender",
+        "description": "Sex of the participant",
+        "levels": {"male": "male", "female": "female"},
+    },
+    "site": {
+        "original_field_name": "None given - in the case of single site study, the site name is the dataset name",
+        "description": "Site of imaging data collection",
+    },
+    "diagnosis": {
+        "original_field_name": "diagnosis",
+        "description": "Diagnosis of the participant",
+        "levels": {
+            "CON": "CONTROL",
+            "SCHZ": "SCHZ [schizophrenia]",
+            "BIPOLAR": "BIPOLAR [bipolar disorder]",
+            "ADHD": "ADHD",
```

Comment:
I know I just approved a PR with your original format, but after a second thought this is closer to what BIDS intends to do.
```suggestion
            "CON": "control",
            "SCHZ": "schizophrenia",
            "BIPOLAR": "bipolar disorder",
            "ADHD": "Attention deficit hyperactivity disorder (ADHD)",
```

Reply:
Ok cool! I was trying to incorporate the original labels here too, but we don't need to? Just make clear what the labels are?